creating_packages/package_repo.rst

```diff
@@ -3,26 +3,29 @@
 Recipe and sources in the same repo
 ===================================
 
-In the previous package we implemented a ``source()`` method that fetched the source code from github. An alternative approach would be embedding the source code into the package recipe, so it is self-contained and it doesn't require to fetch code from external origins when it is necessary to build from sources.
+In the previous package we implemented a ``source()`` method that fetched the source code from
```

Comment:
Also, maybe leave format (like making multiline) changes to the other PR, as the review forces to re-read everything and compare manually with the old text

Reply:
Many work to revert here. I think this is a good opprotunity to make some formatting. Otherwise it would never be done. I agree it makes it harder to review though...

Reply:
No problem, do not revert it. Just for next time